### DISCUSSION TITLE
Shortcut - Save and create new Entry

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9167] [Ideas]
> Created by: Florian (fw@demodern.de) on 2015/03/13 06:38:29 +0000
> Votes at time of UserVoice import: 6

-----------------------------------

Just like the Apple-S  Save current entry...a save and create new entry shortcut would be awesome

**Comment by RitterKnightCreative** [comment]
+1 for this. Was doing some data entry for a client the other night for a FAQ section, among other things, being able to quickly save and add another would have been super helpful.

Something like Cmd/Control-Shift-S (like save as) or cmd-alt-s would be ideal.
